repository_service_tuf_worker/repository.py

```diff
@@ -1171,39 +1525,69 @@ def _run_online_roles_bump(self, force: Optional[bool] = False):
                 f"{Targets.type} don't use online key, skipping 'Targets' role"
             )
         else:
-            targets: Metadata = self._storage_backend.get(Targets.type)
+            targets: Metadata[Targets] = self._storage_backend.get(
+                Targets.type
+            )
             if force or (targets.signed.expires - today) < timedelta(
                 hours=self._hours_before_expire
             ):
                 self._update_targets_delegations_key(targets)
                 self._bump_and_persist(targets, Targets.type)
                 logging.info("Bumped version of 'Targets' role")
-                snapshot_bump = True
 
         if force:
             # Updating all delegated target roles.
-            timestamp = self._update_timestamp(
-                self._update_snapshot(bump_all=True)
+            self._update_timestamp(
+                self._update_snapshot(bump_all=True, only_target=True)
```

Comment:
Why we don't bump snapshot here?

Reply:
We don't need it. When "bump_all" is used. If there is any change in any delegated role, it automatically bumps if there is any ch snapshot.